.github/workflows/deploy.yml

```
@@ -1,16 +1,25 @@
-name: Deploy to GitHub
+name: Deploy Releases
+
 on:
   push:
-    tags:
-      - '*'
+    tags: ["*"]
 
 jobs:
-  deploy:
+  npm:
+    uses: tree-sitter/workflows/.github/workflows/package-npm.yml@main
+    secrets:
+      NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```

Please format the substitutions using a space for consistency with other workflows and the official conventions (I would prefer that even if the upstream workflows do not, we customize them anyways):

```suggestion
      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```